docassemble/MotionToStayEviction/data/questions/review_screen.yml

```diff
@@ -0,0 +1,386 @@
+
+      Name: ${ showifdef('lower_court_case.judge') }
+  - Edit: |-
+      no_notice_of_appeal_filed
+    button: |
+      **When did you file a Notice of Appeal in the Housing Court?**
+
+      I did not file a Notice of Appeal: ${ word(yesno(no_notice_of_appeal_filed)) }
+
+      Date filed: ${ showifdef('date_notice_of_appeal_filed') }
+  - Edit: |-
+      date_housing_court_denied_motion_to_stay
+    button: |
+      **When did the Housing Court judge deny your Motion to Stay?**
+
+      Date: ${ showifdef('date_housing_court_denied_motion_to_stay') }
+  - Edit: |-
+      housing_court_denying_order_text
+    button: |
+      **Enter the text of the Housing Court judge's order denying your Motion to Stay in the Housing Court.**
+
+      ${ showifdef('housing_court_denying_order_text') }
+  - Edit: |-
+      legal_issues
+    button: |
+      **Select the issues of law in your case.**
+
+      ${ legal_issues.true_values() }
+
+      Other: ${ showifdef('other_legal_issues') }
+  - Edit: |-
+      relief_sought
+    button: |
+      **What are you asking for?**
+
+      ${ showifdef('relief_sought') }
+  - Edit: |-
+      parties_to_be_served
+    button: |
+      **List the name and address (or email address) of each attorney or self-represented party that you will deliver copies of your motion, memorandum, and supporting materials to.**
+
+      ${ showifdef('parties_to_be_served') }
+  - Edit: |-
+      service_date
+    button: |
+      **When will you deliver copies of your motion, memorandum, and supporting materials to the plaintiffs' attorney(s)?**
+
+      Date: ${ showifdef('service_date') }
+  - Edit: |-
+      procedural_history
+    button: |
+      **Describe the events in your Housing Court case in the order they happened.**
+
+      ${ showifdef('procedural_history') }
+  - Edit: |-
+      statement_of_facts
+    button: |
+      **Describe the **facts** of your case.**
+
+      ${ showifdef('statement_of_facts') }
+  - Edit: |-
+      likelihood_of_success
+    button: |
+      **Why is your appeal likely to succeed?**
+
+      ${ showifdef('likelihood_of_success') }
+  - Edit: |-
+      defendant_harm
+    button: |
+      **What harm will you or your dependents suffer if your Motion to Stay is denied?**
+
+      ${ showifdef('defendant_harm') }
+  - Edit: |-
+      plaintiff_harm
+    button: |
+      **What harm will the plaintiff suffer if your Motion to Stay is granted?**
+
+      ${ showifdef('plaintiff_harm') }
+  - Edit: |-
+      public_interest_harm_yesno
+    button: |
+      **Does your Motion to Stay have an effect on the public interest?**
+
+      ${ word(yesno(public_interest_harm_yesno)) }
+
+      What effect would your Motion to Stay have on the **public interest**?: ${ showifdef('public_interest_harm') }
+  - Edit: |-
+      argument_summary
+    button: |
+      **Summarize your argument**
+
+      ${ showifdef('argument_summary') }
+---
+event: |-
```

Comment:
I think this should be `field` or `continue button field`, because with event, there isn't a continue button: the only way back to the rest of the interview is with "Undo", which forces you to undo all of your changes on the table screen.